pages/history.tsx

```diff
@@ -185,35 +543,63 @@ const History = () => {
-                    (a, b) =>
-                      Number(Date.parse(b[1].ui.timestamp).toString(10)) -
-                      Number(Date.parse(a[1].ui.timestamp).toString(10))
-                  )
-                  .map(x => {
-                    return x[0] == -1 ? (
-                      <Transfer
-                        address={state.currentContract ?? ""}
-                        key={(x[1] as any).timestamp as any}
-                        prop={x[1] as any}
-                      />
-                    ) : (
-                      <ProposalCard
-                        contract={contract}
-                        id={x[0]}
-                        key={x[0]}
-                        prop={x[1]}
-                        address={state.currentContract ?? ""}
-                        signable={false}
-                      />
-                    );
-                  })}
+                {filteredProposals.map((x, i) => {
+                  return x[0] == -1 ? (
+                    <div
+                      key={(x[1] as any).timestamp}
+                      className="grid h-16 w-full w-full grid-cols-3 items-center gap-8 rounded border-b border-zinc-900 bg-zinc-800 px-6 py-4 text-white lg:grid-cols-4"
+                    >
+                      <span className="ml-11 justify-self-start font-bold">
+                        Received mutez
+                      </span>
+                      <span
+                        className="font-light text-zinc-300"
+                        style={{
+                          minWidth: "7rem",
+                        }}
+                      >
+                        From: <Alias address={(x[1] as any).sender.address} />
+                      </span>
+                      <span
+                        className="font-light text-zinc-300"
+                        style={{
+                          minWidth: "7rem",
+                        }}
+                      >
+                        Amount: {(x[1] as any).amount} mutez
+                      </span>
+                      <span className="hidden justify-self-end lg:block">
+                        {new Date((x[1] as any).timestamp).toLocaleDateString()}{" "}
+                        -{" "}
+                        {`${new Date(
+                          (x[1] as any).timestamp
+                        ).getHours()}:${new Date(
+                          (x[1] as any).timestamp
+                        ).getMinutes()}`}
+                      </span>
+                    </div>
+                  ) : (
+                    <HistoryCard
+                      id={x[0]}
+                      key={x[0]}
+                      isOpen={!!openedPreview[i]}
+                      onClick={() => {
+                        setOpenPreview(v => ({ ...v, [i]: !(v[i] ?? false) }));
+                      }}
+                      status={x[1].ui.status}
+                      date={new Date(x[1].ui.timestamp)}
```

Comment:
I am thinking maybe using resolved timestamp is making more sense.
Once you resolved proposal, you may want to check it quick on top on of history.
This is nice to have if it's not hard to fix and makes sense.